http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterpSpec.scala

```diff
@@ -449,6 +449,72 @@ class DiskServiceInterpSpec extends AnyFlatSpec with LeonardoTestSuite with Test
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "list disks belonging to self and others, if not filtered by role=creator" in isolatedDbTest {
+    val (diskService, _) = makeDiskService()
+    val userInfo = UserInfo(OAuth2BearerToken(""),
+                            WorkbenchUserId("userId"),
+                            WorkbenchEmail("user1@example.com"),
+                            0
+    ) // this email is white listed
```

Comment:
nit: `white list ` -> `allowed list`

Reply:
will do! for other readers: I went looking for industry conversation around these kinds of changes and the best writeup I found was [this Wired article](https://www.wired.com/story/tech-confronts-use-labels-master-slave/). `deny`/`allow` seems clear enough to me!